joomla/joomla-cms
# Isis and hathor frontend links: Frontend uri scheme can be different from admin

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/8109 .
#### Summary of Changes

This updates Isis and hathor templates frontend links because joomla allows different schemes for admin and frontend site.
#### Testing Instructions
1. Use latest staging.
2. Set the site to "Force SSL" in "Administrator Only" in global config.
3. Notice, that you are in `https` in admin as you selected, but:
   - the frontend preview link (top right) is `https`
   - the "View site" link in (bottom left) of the admin panel is `https`
   -  the "Go to site home page." link in the backend login page is `https` (in isis and hathor templates)
4. Apply the patch.
5. Repeat steps 2 and 3 and you will notice the links are now `http` as they should.
